collect_app/src/main/res/drawable/form_state_finalized.xml

```diff
@@ -1,20 +1,17 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="40dp"
-        android:height="40dp"
-        android:viewportWidth="120"
-        android:viewportHeight="120">
+    android:width="40dp"
+    android:height="40dp"
+    android:viewportWidth="120"
+    android:viewportHeight="120">
 
     <path
-            android:pathData="M60,60m-60,0a60,60 0,1 1,120 0a60,60 0,1 1,-120 0"
-            android:fillColor="@color/deep_purple_500"/>
+        android:pathData="M67.195,26.83h-26.83c-3.688,0 -6.673,2.985 -6.673,6.633L33.66,86.537c0,3.648 2.984,6.634 6.673,6.634L80.61,93.171c3.689,0 6.707,-2.986 6.707,-6.634L87.317,46.732L67.195,26.829zM40.365,86.537L40.365,33.463h23.476L63.841,50.05L80.61,50.05v36.488L40.366,86.538z"
+        android:fillColor="?iconColor"/>
```

Comment:
I think it's best avoid `iconColor` and go for Material Theme attributes like `colorOnSurface` or `colorOnPrimary`. These let us account for the fact that in some color schemes the icon color would differ depending on where it was rendered.

Reply:
That's right but `?iconColor` is used in many places like that so would be better to clear that in a separate pr.

Reply:
Absolutely. It's one of the follow ups to #3777 I'll schedule. It would be nice to not have another one to change in that though!